packages/zowe-explorer/src/trees/dataset/DatasetActions.ts

```diff
@@ -1890,10 +1891,32 @@ export class DatasetActions {
         // Figure out what text we are looking for.
         const searchString = await Gui.showInputBox({ prompt: vscode.l10n.t("Enter the text to search for.") });
         if (!searchString) {
-            Gui.showMessage(vscode.l10n.t("Operation Cancelled"));
             return;
         }
 
+        // Handle various search options
+        const caseSensitiveQuickPick = await Gui.showQuickPick(
+            [
+                {
+                    label: vscode.l10n.t("Case Sensitive"),
+                    description: vscode.l10n.t("Perform the search with case sensitivity"),
+                    iconPath: new vscode.ThemeIcon("case-sensitive"),
+                } as vscode.QuickPickItem,
```

Comment:
It seems that this cast is not required since it does not change the type inferred by `Gui.showQuickPick`, although it does make the type inference look cleaner when hovering over the function. Not requesting any changes for this, but just something I observed when reviewing